src/Components/Comments.jsx

```diff
@@ -55,12 +58,27 @@ const Comments = () => {
 		})
 	};
 
+	const handleDelete = (comment_id) => {
+		setDeleted(true)
+		deleteComment(comment_id).then(()=>{
+			getCommentsByArticleId(article_id)
+				.then((data) => {
+					setDeleted(false)
+					setComments(data.comments)
```

Comment:
doesn't need to be, but could this be done a little more optimistically and with state to save a database call everytime a comment is deleted?